spec/views/admin/enterprises/form/_users.html.haml_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "admin/enterprises/form/_users.html.haml" do
+  let(:enterprise) { build(:enterprise) }
+
+  before do
+    assign(:enterprise, enterprise)
+    admin_user = build(:admin_user)
+    allow(admin_user).to receive(:admin?) { true }
+    allow(view).to receive_messages(
+      f: enterprise_form,
+      spree_current_user: admin_user,
+    )
+  end
+
+  describe "notifications setting" do
+    it "is visible when an enterprise is a distributor" do
+      enterprise.sells = "any"
+
+      render
+
+      expect(rendered).to have_selector("select[name='receives_notifications']")
+    end
+
+    it "is not visible when an enterprise is a only a profile" do
```

Comment:
What if they're a producer though...? Are there no emails in that case? :thinking:

Reply:
I wouldn't be the most familiar with this but I had a look in [wiki/Emails](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Emails) and I couldn't see any obvious emails for enterprises with `sells == 'none'` / `is_distributor == false`

Reply:
12. order cycle producers notification
This is email is for `sells == 'none'` / `is_distributor == false`